packages/data-flow/test/integration/strategy.integration.spec.ts

```
@@ -209,30 +209,14 @@ describe("Orchestrator Integration - Strategy Events Processing", () => {
-                    matchAmountInToken: "999999999999999475712",
-                    originalMatchAmountInToken: "999999999999999475712",
-                },
-            ],
-        };
+        const mockDistribution = [
+            {
+                anchorAddress: "0x7340F1a1e4e38F43d2FCC85cdb2b764de36B40c0",
+                payoutAddress: "0x7340F1a1e4e38F43d2FCC85cdb2b764de36B40c0",
+                amount: "1000000000000000000",
+                index: 0,
+            }
```

ESLint

[error] Insert `,`

(prettier/prettier)